Поправил структуры, работают

sartfromua opened the pull request:
Какие параметры классов нужно изменить/добавить?

sartfromua committed:
Поправил структуры, работают

Какие параметры классов нужно изменить/добавить?

sartfromua committed:
Update structures.c

mecchmatProjects reviewed (changes requested):
look at the comments on previous pull request

mecchmatProjects commented on main.c:
```diff
@@ -0,0 +1,9 @@
+
+int main() {
+    Employee emp;
+    emp.person.birthday.day = 10;
```
try to use functions

mecchmatProjects commented on structures.c:
```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
```
replace this to .h file

mecchmatProjects replied:
#include ".....h"
Also it is better to split each structure in separate file, U can do it later...

mecchmatProjects commented on structures.h:
```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "structures.c"
```
delete #include "structures.c"

here is the place for your functions implementatiions

sartfromua replied:
Tried deleting, main.c isn't working without #include "structures.c" in this header.

mecchmatProjects replied:
becouse U compile it wrong

mecchmatProjects replied:
How are U create project? Do U have Makefile?

mecchmatProjects replied:
read LectC.pdf - last chapter

sartfromua replied:
Codeblocks created project for me, having "project file" in folder

sartfromua committed:
Corrected structures, input and print functions

Немного поправил структуры, есть вопросы: формула для зарплаты работника (придумать?), зарплата Руководителя и его Зама вводится или считается по формуле? в задании написано, что Руководитель и Зам - наследники класса работник, но там есть некие несостыковки в параметрах (это я должен как-то сам чинить или я задание не так понял?), также вопрос по взаимоотншению структур Виріб и Обладнання (что от чего наследуется)?
Запускал input и print, на практике работает.

sartfromua committed:
Update main.c

sartfromua committed:
Перераспределил структуры и функции? 

Структуры Head и Deputy (наследники от Employee по заданию) так должны выглядеть?

sartfromua committed:
Unneeded

mecchmatProjects commented on structures.h:
```diff
@@ -0,0 +1,74 @@
+
+
+
+date inputDate();
```
It is much better than it was.
However U'd better split not by functionality, but by classes: Empoyee.h. Empoyee.c, Product.h, Product.c etc
